src/StoreApi/docs/products.md

```diff
@@ -35,34 +36,35 @@ GET /products?return_attribute_counts=pa_size,pa_color
+| `date_column`        | string  |    no    | When limiting response using after/before, which date column to compare against. Allowed values: `date`, `date_gmt`, `modified`, `modified_gmt`                            |
+| `exclude`            | array   |    no    | Ensure result set excludes specific IDs.                                                                                                                                   |
+| `include`            | array   |    no    | Limit result set to specific ids.                                                                                                                                          |
+| `offset`             | integer |    no    | Offset the result set by a specific number of items.                                                                                                                       |
+| `order`              | string  |    no    | Order sort attribute ascending or descending. Allowed values: `asc`, `desc`                                                                                                |
+| `orderby`            | string  |    no    | Sort collection by object attribute. Allowed values : `date`, `modified`, `id`, `include`, `title`, `slug`, `price`, `popularity`, `rating`, `menu_order`, `comment_count` |
+| `parent`             | array   |    no    | Limit result set to those of particular parent IDs.                                                                                                                        |
+| `parent_exclude`     | array   |    no    | Limit result set to all items except those of a particular parent ID.                                                                                                      |
+| `type`               | string  |    no    | Limit result set to products assigned a specific type.                                                                                                                     |
+| `sku`                | string  |    no    | Limit result set to products with specific SKU(s). Use commas to separate.                                                                                                 |
+| `featured`           | boolean |    no    | Limit result set to featured products.                                                                                                                                     |
+| `category`           | string  |    no    | Limit result set to products assigned a specific category ID.                                                                                                              |
+| `product-taxonomy`   | string  |    no    | Limit result set to products assigned to the term ID of that custom product taxonomy. `product-taxonomy` should be the key of the custom product taxonomy registered.      |
```

Comment:
Nitpicking, but I feel it's odd to list `product-taxonomy` between `category` and `category_operator`.

Can we move `product-taxonomy` above `category` or, even better, after `category_operator`?

And two more questions:

* Should we also document `product-category_operator`?
* And should we make it more explicit that `product-category` is a placeholder? Maybe enclosing it in square brackets or something along these lines?

Reply:
Great ideas, thanks @Aljullu 

[0de1d1d](https://github.com/woocommerce/woocommerce-gutenberg-products-block/pull/6152/commits/0de1d1d812d0c2bc701a5d97dfbe2fd8386e3055) moves the `product-taxonomy` after category operator, encloses it in square brackets and also documents `[product-taxonomy]_operator`.